taskRequest/README.md

```diff
@@ -0,0 +1,101 @@
+# Task Request
+
+## Task Request Object
+
+```
+// TBD
+```
+
+## **Requests**
+
+|                     Route                     |          Description          |
+| :-------------------------------------------: | :---------------------------: |
+|    [GET /taskRequests](#get-taskRequests)     |   Returns all task Reqeusts   |
+|     [PUT /taskRequest](#put-taskRequest)      |    Creates a task Request     |
+|   [PATCH /taskRequest](#patch-taskRequests)   | Approves task request to user |
+| [DELETE /taskRequest/:id](#delete-taskRequst) |      Delete task request      |
+
+## **GET /taskRequests**
+
+Returns all tasks to super user.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  rds-session: `<JWT>`
+- Success Response
+  - Code: 200
+    Content:
+    ```
+    {
+      message: "Task requests returned successfully",
+      taskRequests: [
+        {<task request object>}
+      ],
+    }
+    ```
+- Error Response
+  - Code: 401
+    Content: `{'statusCode': 401, 'error': 'Unauthorized User' }`
+  - Code: 403
+    Content: `{'statusCode: 403, 'error': 'Unauthorized access: user is not super user'}`
+  - Code: 500
+    Content: `{'statusCode: 500, 'error': 'Internal Server Error'}`
+
+## **PUT /taskRequest**
+
+Creates a task request.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Headers**  
+  `Content-Type: 'application/json'`
+- **Body**
+  `{<taskInfo, userInfo>}`
```

Comment:
Here I guess `userInfo` is supposed to be an array of userIds. [Array<userId: string>]